backend/core/templates/core/no_contrib_email/body_no_contrib.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}{% get_current_language as LANGUAGE_CODE %}<!DOCTYPE html>
+<html lang="{{ LANGUAGE_CODE }}">
+  <head>
+    <meta charset="UTF-8">
+    {% include "core/no_contrib_email/_body_style.inc.html" %}
+  </head>
+  <body>
+    {% include "core/no_contrib_email/_body_header.inc.html" %}
+    <p>
+      {% blocktranslate %}Tournesol works thanks to the contributions of many people who volunteer their time to compare
+the videos they've seen on YouTube. We look forward to your contributions, if you have the time of
```

Comment:
"you have time" sounds more correct than "you have the time"
```suggestion
the videos they've seen on YouTube. We look forward to your contributions, if you have time of
```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1884/commits/ab14ec91055069e060111305331e8c7fa6d5d669